### ISSUE TITLE
Plugin Browser: Search placeholder feels a tad bit too small

### ISSUE THREAD

**Issue by cpapazoglou** [Automattic/wp-calypso#58293]
### Details

![SS 2021-11-19 at 15 55 22](https://user-images.githubusercontent.com/12430020/142633981-a7625086-be9b-43a5-ad6e-1e83afd0f282.png)

### Related

pdh6GB-6h-p2#comment-315

**Comment by SaxonF** [comment]
@cpapazoglou as we discussed in our conversation yesterday we are trying to put more emphasis on our system this year. While we are revisiting the search placeholder it might also be a good time to revisit the page breadcrumbs so that they align with other parts of Dotcom (e.g. use the same component). Once small change you'll notice is that all breadcrumbs are the same size.

![image](https://user-images.githubusercontent.com/1072756/148893530-8c539d1a-b5e1-4dc4-800e-dc66ea2a66df.png)

In my journey through Figma I've also come across font sizes `10,12,13,14,16,18, 20 + above`. We are considering simplifying to:
```
12px caption
14px base font size
16px heading and emphasising body text
20px page sub-header
32px display small (mostly Recoleta)
44px display large (always Recoleta)
```

I'm not sure how to begin utilising/promoting this font size scale in code but I'd love your help in to ensure all teams should always use this central scale and almost never hard code font size values. The same can be said for spacing (margin and padding), colors, shadows, border radius, font family and other low level tokens.

**Comment by ianstewart** [comment]
It would be good to add the old "learn more" explainer back as well. Example of how this works now with the Domains screen.

![image](https://user-images.githubusercontent.com/545779/148991450-be7cf030-c5e6-4bed-996d-daae64e5c9aa.png)

**Comment by cpapazoglou** [comment]
> @cpapazoglou as we discussed in our conversation yesterday we are trying to put more emphasis on our system this year. While we are revisiting the search placeholder it might also be a good time to revisit the page breadcrumbs so that they align with other parts of Dotcom (e.g. use the same component). Once small change you'll notice is that all breadcrumbs are the same size.
> 
> ![image](https://user-images.githubusercontent.com/1072756/148893530-8c539d1a-b5e1-4dc4-800e-dc66ea2a66df.png)
> 
> In my journey through Figma I've also come across font sizes `10,12,13,14,16,18, 20 + above`. We are considering simplifying to:
> 
> ```
> 12px caption
> 14px base font size
> 16px heading and emphasising body text
> 20px page sub-header
> 32px display small (mostly Recoleta)
> 44px display large (always Recoleta)
> ```
> 
> I'm not sure how to begin utilising/promoting this font size scale in code but I'd love your help in to ensure all teams should always use this central scale and almost never hard code font size values. The same can be said for spacing (margin and padding), colors, shadows, border radius, font family and other low level tokens.

@SaxonF one note to the above font sizes is that `input` items like the search box can't have `font-size < 16px`. At least in ios, ipados breakpoints using anything less than 16px will result in the web page being zoomed in when focusing on that input.

**Comment by lsl** [comment]
@SaxonF this needs to be done at a higher level than marketplace work.

Happy to create a PR (or two) that modifies [the definitions](https://github.com/Automattic/wp-calypso/blob/5d95d41594689f75cf0d5afc24632e96520faafe/packages/typography/styles/variables.scss#L46) we already have but I think it would need wider review (i18n, a11y and marketing come to mind.)

For this issue, I suggest we:

- [ ] Change the "Try searching "ecommerce"" [placeholder font-size](https://github.com/Automattic/wp-calypso/blob/831b4c593e979f6b214724ff4693c2aebbc9fa26/client/my-sites/plugins/plugins-browser/style.scss#L18) to `font-body-small` from `font-body-extra-small`.

**Comment by SaxonF** [comment]
Sounds good @lsl . We're going to revisit search in future iterations anyway to put more emphasis on it.

**Comment by lupus2k** [comment]
Closed by #61135
